src/modules/common/components/uploadJSONInput/uploadJSONInput.js

```diff
@@ -22,22 +23,26 @@ const UploadJSONInput = ({
     }
   };
 
-  const onInputChange = ({ target }) => {
-    const file = target.files[0];
+  const onInputChange = useCallback((acceptedFiles) => {
+    const file = acceptedFiles[0];
 
     const reader = new FileReader();
     reader.onload = (event) => {
       onParse(event.target.result);
     };
     reader.readAsText(file);
-  };
+  }, []);
 
   const onPaste = (event) => {
     onParse(event.clipboardData.getData('text'));
   };
 
+  const { getRootProps, getInputProps, isDragActive } = useDropzone({
+    onDrop: onInputChange,
```

Comment:
useDropzone  takes this option

```
    accept: {
      'application/JSON': ['.json'],
    },
```

We do only accept json for this feature, correct?

Reply:
We're passing the same option directly to the input field which is a similar thing. We also have a verification regardless of which file type is passed and we throw an error if the file is invalid or an invalid JSON

Reply:
Yes for the input we are passing `accept="application/JSON"`
However, this is what i see:
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/3320483c-40dd-42e4-8776-aa435a756228)

If we add my suggested code above, it would look like this, which is better UX
![image](https://github.com/LiskHQ/lisk-desktop/assets/8784876/d7593be7-067f-4c78-b392-a06f1fa040ff)

Reply:
Fixed